devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy

```diff
@@ -4,14 +4,12 @@
  *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
  *  in compliance with the License. You may obtain a copy of the License at:
  *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ *		http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Not sure some of the indentation changes are needed.

Reply:
@dkirker ok, I will revert that.

Reply:
It looks like one of the newer commits put even more tabs in there.

Reply:
ok, fixed